features/logging/4.x/elasticsearch.feature

```diff
@@ -0,0 +1,65 @@
+@clusterlogging
+Feature: Elasticsearch related tests
+
+  # @auther qitang@redhat.com
+  # @case_id OCP-22050
+  @admin
+  @destructive
+  Scenario: Elasticsearch using dynamic volumes
+    Given I switch to cluster admin pseudo user
+    And default storageclass is stored in the :default_sc clipboard
+    Given I delete the clusterlogging instance
+    Then the step should succeed
+    And I run the :create client command with:
+      | f | https://raw.githubusercontent.com/openshift-qe/v3-testfiles/master/logging/clusterlogging/example_unmanaged.yaml |
+    Then the step should succeed
+    Given I use the "openshift-logging" project
+    Given I wait for the "instance" clusterloggings to appear
+    Given I get project clusterlogging named "instance" as YAML
+    Then the step should succeed
+    And the output should contain:
+      | managementState: Unmanaged |
+    Given I get project elasticsearch named "elasticsearch" as YAML
+    Then the step should fail
+    When I run the :patch client command with:
+      | resource      | clusterlogging                                                                                        |
+      | resource_name | instance                                                                                              |
+      | p             | {"spec":{"logStore":{"elasticsearch":{"storage":{"storageClassName": "<%= cb.default_sc.name %>"}}}}} |
+      | type          | merge                                                                                                 |
+    Then the step should succeed
+    Given I get project clusterlogging named "instance" as YAML
```

Comment:
looking at how many times we look at raw YAML, perhaps create a clusterlogging class? @pruan-rht ?